exe-unit/src/runtime.rs

```diff
@@ -53,7 +53,7 @@ impl RuntimeArgs {
         let mut mem_gib = None;
         let mut storage_gib = None;
         if with_inf {
-            cpu_cores = agreement.infrastructure.get("cpu.cores").cloned();
+            cpu_cores = agreement.infrastructure.get("cpu.threads").cloned();
```

Comment:
we are mixing up cores and threads, but we have more important issues, so let it be

Reply:
yes, requestors currently demand logical threads not physical cores